docs/src/main/tut/docs.md

```diff
@@ -31,13 +31,13 @@ To begin, add the following dependency to your SBT build file:
 [comment]: # (Start Replace)
 
 ```scala
-"com.47deg" %% "fetch" % "1.2.2"
+"com.47deg" %% "fetch" % "@VERSION@"
```

Comment:
does the plugin work with tut too?

Reply:
Nop, but in my next PR I'll migrate from tut to mdoc.

> (still pending to do the same for the docs, but I'm planning to do in a separate PR).

Reply:
ah yes sorry did not see that :+1: